### DISCUSSION TITLE
Discord Community Server

### DISCUSSION THREAD

**Discussion by patrys** [saleor/saleor#13208] [Announcements]
Hello everyone,

We hear that GitHub Discussions are suitable for support but not great for conversations, so we've launched a Discord community server for Saleor.

We're also starting a series of live community updates we want to hold every two weeks on Discord.

Here's the link to join: https://saleor.io/discord

**Comment by slamer59** [comment]
Hi, Invitation is invalid now.

**Reply by timuric** [reply]
I just tested and it works, could you please try again?

**Reply by slamer59** [reply]
Mmm strange. It wasnt on my professionnal computer... Thanks !
